internal/server/network/driver_ovn.go

```diff
@@ -5757,7 +5797,20 @@ func (n *ovn) LoadBalancerCreate(loadBalancer api.NetworkLoadBalancersPost, clie
 
 		err = n.state.DB.Cluster.Transaction(context.TODO(), func(ctx context.Context, tx *db.ClusterTx) error {
 			// Create load balancer DB record.
-			loadBalancerID, err = tx.CreateNetworkLoadBalancer(ctx, n.ID(), memberSpecific, &loadBalancer)
+			node_id := sql.NullInt64{
+				Valid: memberSpecific,
+				Int64: tx.GetNodeID(),
+			}
+			network_id := n.ID()
+			lb := dbCluster.NetworkLoadBalancer{
+				NetworkID: network_id,
+				NodeID: node_id,
+				ListenAddress: loadBalancer.ListenAddress,
+				Description: loadBalancer.Description,
+				Backends: loadBalancer.Backends,
+				Ports: loadBalancer.Ports,
```

Comment:
I think you should check these two fields if they're null or not, and assign Backends/Ports to []. Otherwise, I believe this messes up the JSON marshalling and unmarshalling.